ds/gdb.go

```diff
@@ -56,6 +56,97 @@ func (g *GDB) getID(p string) (string, error) {
 	return id, nil
 }
 
+type ImageTypeName string
+
+func bucketImagesByType(images []gdb.ParsedGameImage) map[ImageTypeName][]gdb.ParsedGameImage {
+	res := make(map[ImageTypeName][]gdb.ParsedGameImage)
+
+	for _, image := range images {
+		imageType := ImageTypeName(image.Type)
+		existing := res[imageType]
+
+		existing = append(existing, image)
+		res[imageType] = existing
```

Comment:
This could be simplified to:
res[imageType] = append(res[imageType], image)

Reply:
Fixed